nntrainer/graph/graph_core.h

```diff
@@ -290,6 +294,14 @@ class GraphCore {
    */
   void
   makeAdjacencyList(std::vector<std::list<std::shared_ptr<GraphNode>>> &adj);
+
+  /**
+   * @brief     Get index of the node with given name
+   *
+   * @param     name Name of the node
+   * @return    internal index of the node
+   */
+  unsigned int getNodeIdx(const std::string &name);
```

Comment:
We still have the index but it's accessible from the name, am I correct?

Reply:
the index is only for internal usage. The index is an index given to a node, but index to the location where the node exists in the structure (adjacency list in this case).

Reply:
I see